src/cpu/jit_uni_eltwise.cpp

```diff
@@ -724,108 +724,59 @@ template struct jit_uni_eltwise_injector_f32<sse41>;
 
 
 struct jit_args {
-    const float *from;
-    const float *for_comparison;
-    const float *to;
+    const void *from;
+    const void *for_comparison;
+    const void *to;
     size_t work_amount;
 };
 
-struct jit_uni_eltwise_kernel_f32 : public c_compatible {
+struct jit_uni_eltwise_kernel : public c_compatible {
+    enum {
+        _rnd_trunc = 3u // Round toward zero
```

Comment:
`jit_uni_eltwise_kernel` is a base class for both `jit_uni_relu_kernel` and `jit_uni_kernel_fwd` kernel. 

The original intention for  `jit_uni_eltwise_kernel` struct is to be just an interface devoid of any implementation level details. I think it's better to hide all implementation details in where they are actually used i.e. `jit_uni_relu_kernel` in this case.